apiary.apib

```diff
@@ -64,29 +90,45 @@ Destroying an account, and changing account properties
 like username and password, require to pass the user's
 password as `x-password` header.
 
-### Sign Up [PUT]
+### Sign Up [POST]
```

Comment:
Why would you use POST instead of PUT? As far as I know, you use PUT when you know the location of the resource, which is `/session/account` in that case, and use POST if you don't, e.g. `POST /accounts`